packages/ui-components/src/components/range/range.tsx

```diff
@@ -38,6 +44,14 @@ export class KvRange implements IRange, IRangeEvents {
 		return parseInt(element.value);
 	};
 
+	private getMinLabel = (): string => {
```

Comment:
can we change these to `get` properties? this would avoid the calls to this function on line 78 and 79